pkg/api/store/cluster/cluster_store.go

```diff
@@ -128,6 +128,10 @@ func validateNetworkFlag(data map[string]interface{}) error {
 		values.PutValue(data, false, "enableNetworkPolicy")
 	} else if value := convert.ToBool(enableNetworkPolicy); value {
 		if rkeConfig == nil {
+			if create {
+				values.PutValue(data, false, "enableNetworkPolicy")
```

Comment:
@alena1108 my concern is when creates cluster using api and actually passes value true (not reading from schema defaults), they won't get any error and we'll silently make it off.

Reply:
@kinarashah it can be handled separately as this was the issue even before. The solution would be to validate on the api level, and raise an error if true is passed

Reply:
@kinarashah I don't think of the way to deal with that better. Like you'd never know if it's something user inputed, or modified by the default setting.